browser/src/ClinvarVariantsTrack/ClinvarVariantTrack.tsx

```diff
@@ -87,6 +87,22 @@ const SelectCategoryButton = styled(Button)`
   line-height: 18px;
 `
 
+const FilterRow = styled.div`
+  display: flex;
+  align-items: center;
```

Comment:
Does it look a bit cleaner to use `justify-content: space-between;` here? This puts the review status filter justified to the right. Otherwise I feel like there's not enough space between this and "Only show Clinvar..."

Thoughts @ch-kr @rileyhgrant @phildarnowsky?

![image](https://github.com/broadinstitute/gnomad-browser/assets/8867295/43b0878c-8559-40e2-aadd-5a622e86abbe)

Reply:
I actually like this layout better I feel like it similar to the Coverage track with the right side dropdown

Reply:
I prefer this as well. also, it'll be cool to have this feature on the browser!